backend/src/test/java/ai/verta/modeldb/FindProjectEntitiesTest.java

```diff
@@ -2325,29 +2323,19 @@ public void findAndSortExperimentRunsByMetricsTest() {
   public void findExperimentRunsNegativeTest() {
     LOGGER.info("FindExperimentRuns Negative test start................................");
 
-    FindExperimentRuns findExperimentRuns;
-
-    try {
-      findExperimentRuns = FindExperimentRuns.newBuilder().setProjectId("12321").build();
-      experimentRunServiceStub.findExperimentRuns(findExperimentRuns);
-      fail();
-    } catch (StatusRuntimeException ex) {
-      checkEqualsAssert(ex);
-    }
+    FindExperimentRuns findExperimentRuns =
+        FindExperimentRuns.newBuilder().setProjectId("12321").build();
+    FindExperimentRuns.Response response =
+        experimentRunServiceStub.findExperimentRuns(findExperimentRuns);
+    assertEquals("Expected response not found", 0, response.getExperimentRunsCount());
 
-    try {
-      List<String> experimentRunIds = new ArrayList<>();
-      experimentRunIds.add("abc");
-      experimentRunIds.add("xyz");
-      findExperimentRuns =
-          FindExperimentRuns.newBuilder().addAllExperimentRunIds(experimentRunIds).build();
-      experimentRunServiceStub.findExperimentRuns(findExperimentRuns);
-      fail();
-    } catch (StatusRuntimeException exc) {
-      Status status = Status.fromThrowable(exc);
-      LOGGER.warn("Error Code : " + status.getCode() + " Description : " + status.getDescription());
-      assertEquals(Status.PERMISSION_DENIED.getCode(), status.getCode());
-    }
+    List<String> experimentRunIds = new ArrayList<>();
+    experimentRunIds.add("abc");
+    experimentRunIds.add("xyz");
+    findExperimentRuns =
+        FindExperimentRuns.newBuilder().addAllExperimentRunIds(experimentRunIds).build();
+    response = experimentRunServiceStub.findExperimentRuns(findExperimentRuns);
+    assertEquals("Expected response not found", 0, response.getExperimentRunsCount());
```

Comment:
Now we are checking permission first so if given project id is not valid it will return empty list instead of PERMISSION_DENIED